### ISSUE TITLE
+combat/spar doesn't work for non-staff

### ISSUE THREAD

**Issue by lynnfaraday** [lynnfaraday/MUSH#28]
The +combat/spar command doesn't work right if used by a non-staff-member.  This is a permissions issue.

---

No matter what combat is started, of any type, it alway gets the number #640.

When +combat/spar is done and an attack is successful receive: <COMBAT> Jerric attacks Pawel with Sparring - Serious Stun wound to Abdomen.#-1 PERMISSION DENIED

In +combat/spar when attempting to leave the combat receive: <COMBAT> #-1 PERMISSION DENIEDYou leave the combat.
 The player's attribute Combat remains set at #640 and can't join another combat till the attribute is cleared manually
 When attempting to +Combat/leave again receive: <COMBAT> #-1 INVALID OBJECT#-1 INVALID OBJECTYou leave the combat.
 Even when the spar is stopped the initiator of the spar the attribute Combat remains set at #640. 

+combat/start real works fine and when the player leaves the combat attribute is cleared. When the combat is stopped it also clears the attribute.

In summary, every combat (either real or spar) instance is always set to #640.
In SPAR the player receives those error messages and the combat attribute remains set to #640.
